test/fuzz-test.js

```diff
@@ -0,0 +1,80 @@
+var assert = require('assert')
```

Comment:
Interesting,  I'm not sure how useful this is, since it is a non-deterministic test,  but I'll give it a further read through later.